.repoconfig

```diff
@@ -3,22 +3,23 @@ repoconfig=5
 
 [format]
 path-ignore=<<EOT
-# rooted folder paths
+# specially named folders that will be matched anywhere in a path (e.g. all files under any 'imgs' will be ignored)
 (^|/)(
  .github
 |.yamato
 |artifacts
 |imgs
 |LocalTestProjects
 |TestProjects
-|Tools
 |com\.unity\.testing\..*
 )(/|$)
 |
+# rooted folder paths
+^(
+ Tools
+)
```

Comment:
as per https://github.cds.internal.unity3d.com/unity/unity/blob/255ba125ce3476a5e6e6881bef287aebad83539a/.repoconfig#L29

You should apply this to make sure further updates won't mess up the regex.

```suggestion
)/|
```

N.B.: I haven't tested this myself but it should work (famous last words)